appinventor/components-ios/src/LinearView.swift

```diff
@@ -263,6 +263,24 @@ public class LinearView: UIView {
     }
   }
 
+  open func resetView(){
+    /* Resets the all the width*/
+    widthFillParent = 0
+    widthFillParentConstraint?.isActive = false
+    widthFillParentConstraint = nil
+    _innerEqualConstraint.isActive = true
+    _innerHeadZero.isActive = false
```

Comment:
It seems like the default should be for the head zero to be true and equal constraints to be false so that this aligns-head.